dashboard/lib/policies/inline_answer.rb

```diff
@@ -11,6 +10,10 @@ def self.visible_for_script_level?(user, script_level)
     script = script_level.try(:script)
     return true if visible_for_unit?(user, script)
 
+    # In PL Courses if a level is marked as an instructor in training level and the user
+    # can be a participant in the course and is a verified instructor show the instructor only content
```

Comment:
This sentence is a bit hard to digest. Thanks to the effort you've put into naming, the code itself is quite readable, so I would suggest simply removing this comment. if the reader wants more detail, they can click through to the definition of view_as_instructor_in_training?, which already explains most of what you are saying here: https://github.com/code-dot-org/code-dot-org/blob/70357c6d871da3501bc6f8cfe6fccacd17c0c8ca/dashboard/app/models/script_level.rb#L669-L674

Reply:
Removed the comment. Thanks!